terraform/modules/tf_cloudtrail/main.tf

```diff
@@ -1,17 +1,17 @@
 locals {
-  apply_filter_string    = "{ $$.awsRegion != \"${var.region}\" }"
+  apply_filter_string    = "{ $.awsRegion != \"${var.region}\" }"
```

Comment:
I actually thing this might be a bad change here - we probably need the double dollar signs

Reply:
OK bringing it back